### DISCUSSION TITLE
Issue with EML (html?) from Paperless - gotenberg returning 503

### DISCUSSION THREAD

**Discussion by psyciknz** [gotenberg/gotenberg#928] [Q&A]
As part of paperless, it has specified that gotenberg be used.   Every email that I am getting it to pass to gotenberg is getting me a 503 error.   The gotenberg logs have the following:
```
convert HTML to PDF: convert PDF: convert PDF to '{PdfA:PDF/A-2b PdfUa:false}' with multi PDF engines: convert PDF to '{PdfA:PDF/A-2b PdfUa:false}' with ExifTool: method not supported; convert PDF to '{PdfA:PDF/A-2b PdfUa:false}' with LibreOffice: process first start: start process: LibreOffice process exited: wait for unix process: signal: killed; convert PDF to '{PdfA:PDF/A-2b PdfUa:false}' with pdfcpu: method not supported; convert PDF to '{PdfA:PDF/A-2b PdfUa:false}' with PDFtk: method not supported; convert PDF to '{PdfA:PDF/A-2b PdfUa:false}' with QPDF: method not supported
```

My current docker-compose command to start gotenbery is the following:
```
command:
      - "gotenberg"
      - "--chromium-disable-javascript=true"
      - "--chromium-allow-list=file:///tmp/.*"
      - "--api-timeout=800s"
```

Any guidance?

**Comment by gulien** [comment]
Hello @psyciknz,

` LibreOffice: process first start: start process: LibreOffice process exited`

Looks like failed to start 🤔 try with the flag `--libreoffice-auto-start`

**Reply by psyciknz** [reply]
If I'm just converting emails (which is just html and pdf). Does it need libreoffice?

And if negative is there something int he command to stop it starting?

**Reply by gulien** [reply]
Yep it does to handle the PDF/A conversion (Chromium handles only the HTML => PDF part).

**Reply by psyciknz** [reply]
Ok so it looks like it's started now.

[SYSTEM] modules: api chromium exiftool libreoffice libreoffice-api libreoffice-pdfengine logging pdfcpu pdfengines pdftk prometheus qpdf webhook 
[SYSTEM] pdfengines: exiftool libreoffice-pdfengine pdfcpu pdftk qpdf
[SYSTEM] prometheus: collecting metrics
[SYSTEM] chromium: Chromium ready to start
[SYSTEM] libreoffice-api: LibreOffice automatically started
[SYSTEM] api: server listening on port 3000

**Reply by psyciknz** [reply]
Awesome.

Thanks for that.  I wonder if I need to let the paperless people know that. 

Or could it just have been a peculiarity on my side?

**Reply by gulien** [reply]
It might be related to your system indeed: not enough resources maybe?

**Reply by gulien** [reply]
Like for instance Chrome + LibreOffice is too much

**Reply by psyciknz** [reply]
I did originally have a memory restriction in place.  But had upped that and recreated container
